pages/common/tlmgr-recreate-tlpdb.md

```diff
@@ -0,0 +1,9 @@
+# tlmgr recreate-tlpdb
+
+> Recreate the TeX Live package database.
+> This command has no documentation and therefore should probably not be used by humans.
```

Comment:
```suggestion
> This command should not be invoked by the user. 
```

Reply:
Well, the thing is, I don't know if it shouldn't. I just assumed it, because there was absolutely no documentation, except for the source code itself.

Reply:
I think this command exists to be used under certain circumstances, also there is a `--platform` argument.
 ```suggestion
```

Reply:
I wouldn't be so sure about that. There are even debug outputs in the source code. Also, this command has to be run as root, which can become risky if it isn't properly documented.

Reply:
I just looked around for this command, and I found some threads where this command is the answer to some errors.

Reply:
How about
```suggestion
> This command should be used with caution, because it is poorly documented.
```

Reply:
That looks like a good suggestion, @marchersimon